R/utils-bin.R

```diff
@@ -1,20 +1,24 @@
-' Binning
+#' Binning
 #' 
 #'
 #' This function divides the range of ‘x’ into intervals and codes 
 #' the values in ‘x’ according to which interval they fall
 #' @param x Numeric or Date vector to bin
 #' @param binWidth number to increment bin bounds by, or string for dates ex: 'month'
-#' @return Character vector of coded values 
+#' @param viewport List of min and max values to consider as the range of data
+#' @param stringsAsFactors logical indicating whether a factor or character vector should be returned
+#' @return Character vector (or factor) of coded values 
```

Comment:
Out of curiosity, when would we want factors instead of strings? So far we've kept factors out of the repo. Are you foreseeing a use for them soon?

Reply:
Ohh because of this line? `overlayValues <- levels(.pd[[x]])`
And because we want to know about bins that may not have any data in them. That's why we can't do `unique(.pd[[x]])`. Okay cool :)